.github/workflows/ci.yml

```diff
@@ -174,7 +174,9 @@ jobs:
     needs: [pre-commit, test-package-python]
     uses: ./.github/workflows/app-build-verify.yml
     with:
-      python-version: "3.9"
+      # This *must* be the version of Python that is the system Python on ubuntu-latest.
+      # Otherwise native system package builds won't be included.
+      python-version: "3.10"
```

Comment:
Completely agreed it's unstable. However, it's unstable on a 2-year cadence, with the next update not expected until mid to late 2025 if history is any indication. I think that's a cadence where manual processes are acceptable.

Reply:
Right....but the period of instability may not be that short. The rollout of the new ubuntu version to all runners can take weeks or probably even longer. So, sometimes the workflow passes and other times it will fail during the transition.

Reply:
As a reference, GitHub expected the transition for `ubuntu-latest` to `ubuntu-22.04` last year to take 8 weeks.

https://github.blog/changelog/2022-11-09-github-actions-ubuntu-latest-workflows-will-use-ubuntu-22-04

Reply:
Hrm... hadn't thought of that. I guess that means `ubuntu-latest` can't be used for the CI checks that are verifying local system builds.